.github/workflows/build-notebooks-TEMPLATE.yaml

```
@@ -43,6 +43,14 @@ jobs:
 
           df -h
 
+          sudo apt-get update
+          sudo apt-get remove -y '^dotnet-.*'
+          sudo apt-get remove -y '^llvm-.*'
+          sudo apt-get remove -y 'php.*'
+          sudo apt-get remove -y '^mongodb-.*'
+          sudo apt-get autoremove -y
+          sudo apt-get clean
```

Is this necessary? it adds more than 30 seconds to every job run, and given how many jobs we have, it adds up. Maybe run single `sudo apt-get remove -y` command that removes all these?

```suggestion
          sudo apt-get update
          sudo apt-get remove -y '^dotnet-.*' '^llvm-.*' 'php.*' '^mongodb-.*'
          sudo apt-get autoremove -y
          sudo apt-get clean
```

---

It came out during my investigation of the full storage. These lines free around 7GB.
I can remove them as they are not part of the solution anymore but I figure it wouldn't hurt to keep them.